www-servers/caddy/caddy-2.10.2-r1.ebuild

```diff
@@ -14,10 +14,11 @@ if [[ "${PV}" == 9999* ]]; then
 else
 	SRC_URI="
 		https://github.com/caddyserver/caddy/archive/v${PV}.tar.gz -> ${P}.tar.gz
-		https://github.com/rahilarious/gentoo-distfiles/releases/download/${P}/deps.tar.xz -> ${P}-deps.tar.xz
+		https://github.com/rahilarious/gentoo-distfiles/releases/download/${PF}/deps.tar.xz -> ${PF}-deps.tar.xz
```

Comment:
In a followup, please drop the use of `${PF}` here. It's a nuisance if someone needs to revbump the ebuild for any other reason (e.g. dep fixes).

You may want to use `_pN` for this instead.